MediaRecorder.html

```diff
@@ -326,11 +329,11 @@
             playable, but the combination of all the Blobs from a completed
             recording MUST be playable.</p>
 
-            <p> If any Track within the MediaStream is muted at any time (i.e.,
-            if its <code>muted</code> flag is set to <code>true</code>), the UA
-            MUST insert black frames or silence until the Track is unmuted, or
-            record nothing, depending on the value of
-            <a>ignoreMutedMedia</a>.</p>
+            <p> The UA MUST insert black frames or silence, or record nothing,
+            depending on the value of <a>ignoreMutedMedia</a>, for as long as
+            a given <var>track</var> within the MediaStream  is
+            <a href="https://www.w3.org/TR/mediacapture-streams/#widl-MediaStreamTrack-muted">
+            muted</a>. This behaviour only applies to the muted tracks.</p>
 
```

Comment:
In a multitrack recording, is it possible to record nothing on one of the tracks?
I can't figure out what it means in terms of timestamps; if track A is recorded and track B is muted for 10 seconds with ignoreMutedMedia=true, will they then be 10 seconds out of sync?

WRT setting it, I was thinking of "disabled", which is JS-settable. Is "disabled" interacting with recording at all, or does it just cause blackness?